### PR TITLE
Cors headers

### PR DESCRIPTION
**Pull request by akintner**
This PR sets the CORS accessibility for the API.

@Dpalazzari @wlffann @case-eee

### CODE REVIEW

File: app/controllers/api/v1/base_controller.rb [hunk 1]
```diff
@@ -5,15 +5,15 @@ class API::V1::BaseController < ApplicationController
...
+    headers['Access-Control-Allow-Methods'] = 'GET, POST'
     headers['Access-Control-Allow-Headers'] = 'Origin, Content-Type, Accept, Authorization, Token'
     headers['Access-Control-Max-Age'] = "1728000"
   end
 
   def cors_preflight_check
     if request.method == 'OPTIONS'
       headers['Access-Control-Allow-Origin'] = '*'
-      headers['Access-Control-Allow-Methods'] = 'GET'
+      headers['Access-Control-Allow-Methods'] = 'GET, POST'
```
Comment (Dpalazzari) [inline app/controllers/api/v1/base_controller.rb]: Nice. A problem for later solved right now.

Comment (akintner) [inline app/controllers/api/v1/base_controller.rb]: turned out it was super, duper simple

**Comment by Dpalazzari** [comment]
@akintner I believe in your code. Will merge. If we get some strange errors later, I'll know who to call. 👍

**Comment by case-eee** [comment]
@akintner @Dpalazzari 👍

**Pull request merged**